recipes/torchaudio/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.8.0" %}
+{% set pytorch_version = "1.8.0" %}
```

Comment:
pytorch 1.8 is very likely broken at conda-forge....

It isn't that I don't want to get your recipe in. but I think that it should target the latest pinning of pytorch, 1.12 and very likely 1.13 (soon).

if this is for you trying to get your work done, I would really suggest just uploading to your own channel. 


Once the recipe is ready for wider distribution, then you can revive it here and others can benefit, but at this stage, we would be releasing software on day 1, with no real path to installation.